vortex-vector/src/binaryview/vector.rs

```diff
@@ -185,13 +185,7 @@ impl<T: BinaryViewType> VectorOps for BinaryViewVector<T> {
             }
         };
 
-        let buffers_mut = match Arc::try_unwrap(self.buffers) {
-            Ok(buffers) => buffers.into_vec(),
-            Err(buffers) => {
-                // Backup: collect a new Vec with clones of each buffer
-                buffers.iter().cloned().collect()
-            }
-        };
+        let buffers_mut = self.buffers.to_vec();
```

Comment:
The reason we had `Arc<Box<[..]>>` was because `Box<[..]>::into_vec` is zero-copy. This now _always_ creates a new `Vec<..>`

Not sure it makes a huge difference. And admittedly wasn't made obvious in the code

Reply:
Yea I was just annoyed at the cruft required to construct this type and thought the extra Vec allocation wouldn't be too harmful, but I can revert this